src/ext-test/java/org/opentripplanner/ext/fares/impl/OrcaFareServiceTest.java

```diff
@@ -440,17 +415,17 @@ void calculateCashFreeTransferKCMetro() {
   @Test
   void calculateTransferExtension() {
```

Comment:
We relied on Sound Transit Link fares to get several different values for this test, so I had to find other combinations of routes to get three increasing fares for this test.